build.gradle

```diff
@@ -169,7 +175,7 @@ tasks.register('integrationTest', Test) {
 }
 
 dependencies {
-    api platform("com.exactpro.th2:bom:4.0.2")
+    api platform("com.exactpro.th2:bom:4.0.3")
```

Comment:
Also, please remove 
implementation 'org.apache.logging.log4j:log4j-1.2-api'
After that we should change logging configuration.
https://github.com/th2-net/th2-common-j/blob/dev-version-5/src/main/java/com/exactpro/th2/common/schema/util/Log4jConfigUtils.kt#L38